internal/character/natasha/skill.go

```diff
@@ -0,0 +1,68 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Skill key.Modifier = "natasha-skill"
+)
+
+func init() {
+
+	//Nat HOT from skill
+	modifier.Register(
+		Skill,
+		modifier.Config{
+			Duration:          2,
+			Stacking:          modifier.Unique,
+			StatusType:        model.StatusType_STATUS_BUFF,
+			CanModifySnapshot: true,
+			Listeners: modifier.Listeners{
+				OnPhase1: func(mod *modifier.Instance) {
+					char, _ := mod.Engine().CharacterInfo(mod.Source())
+					mod.Engine().Heal(info.Heal{
+						Source:    mod.Source(),
+						Targets:   []key.TargetID{mod.Owner()},
+						BaseHeal:  info.HealMap{model.HealFormula_BY_HEALER_MAX_HP: skillContinuous[char.SkillLevelIndex()]},
+						HealValue: skillContinuousFlat[char.SkillLevelIndex()],
+					},
+					)
+				},
+			},
+		},
+	)
+}
+
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
```

Comment:
Skill logic should be the following in order:
1. If A2 is enabled, dispel debuff on passed in target
2. Perform heal
3. Add modifier (where duration is 2 or 3 depending on if A6 is enabled)
4. Add 30 energy to Natasha